tests/flytekit/unit/annotated/test_serialization.py

```diff
@@ -204,20 +204,20 @@ def my_wf(a: int, b: str) -> (int, str):
 def test_serialization_branch():
     @task
     def mimic(a: int) -> typing.NamedTuple("OutputsBC", c=int):
-        return a
+        return (a,)
```

Comment:
Such a weird syntax, comma and paranthesis

Reply:
This is a standard Python syntax for forcing a tuple construction...